### DISCUSSION TITLE
Pentair GloBrite Help Needed

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#978] [Q&A]
I have 4 Pentair GloBrite lights in my pool (thanks to the installer for putting in crap!).  Anyway, they're acting badly but I don't believe the lights  themselves are broken at the moment.  Here's what's going on...

One of the lights periodically refuses to turn on.  On a  few occasions, I've duplicated my SD card from a full backup & then the light works fine... for a few  days.  I've checked the wire nuts in the lights' transformer box, tightened them a bit, but every now & then, no output at all from the light.

I've tried toggling the lights on/off, changing colors, tried light shows, etc., without success.

Last night the problem light from above *and* the light next to it both refused to turn on.  I tried my usual toggles, color &  show changes, etc. without any luck.  I've not yet gone to a backup SD card, but argh!

The lights are not configured as a light group & are all controlled by one relay.  They're configured as shown below in Nixie & REM:

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/b507a02e-1836-48e1-b1d6-0426f9f352d1)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/26928b13-85d3-443a-9f76-ae9591e96c19)

Could the problem be a result of not adding additional time to the on/off delays?  Is there anyone with experience who can suggest what the delays should be?  Is there anything else I can try to get my two problem lights going reliably (other than replacing them with Jandy lights)?

I appreciate any & all troubleshooting & configuration suggestions!

**Comment by tagyoureit** [comment]
Are all 4 lights wired to the same circuit?  How are they wired?

If you turn the light on/off without setting a color or theme, it will just trigger the circuit on/off a single time and the delays will not be in play.  Is there a difference if you toggle the circuit without setting a theme/color?

**Reply by FlaMike** [reply]
Thank you for your reply.

All 4 lights are on one circuit, the transformer is controlled by a 3 hp relay, as it was when the pool was built.  When I converted the ET 4 to a Nixie, I left the wiring intact & just migrated the relay to be controlled by the Sequent relay board.

I have not done the various toggle experiments you mention but I will when both Nixie & REM are up to snuff (I know you've seen my errant Nixie threads.)

I did open the transformer box & ensure the wire nuts form the transformer to the lights were tight & secure--one was slightly loose, but tightening it did not fix the light outages.

The light behavior is somewhat puzzling.  Usually, only 1 light works.  Last night 2 worked.  Rarely, 3 work.  The 4th light may be toast--it flickered on & off for a couple nights & hasn't been illuminated since.  You know how much I enjoy bizzare behavior ;-)

I have tried changing colors, light shows, etc., without seeing any differences in the number of working lights--for any particular night only 1, 2, or 3 lights worked regardless of color, show selection, changes to holds, etc.  Pool lights are just so darned odd critters!  I have also tried grouping the lights & synching them without any affect.

**Reply by rstrouse** [reply]
Don't expect too much out of IntelliBrite.  It is based on old tech born from the old Fiberstars crap.  If I ever get to retire I am going to create a DMX controller for mine.

**Reply by FlaMike** [reply]
I'll be happy to contract you to do something similar here.  You already know what the beaches are like here :-)  I fear the real challenge in my pool is those darned niches for the lights.  Really limits your options, no doubt by design.  Gotta hand it to the IP attorneys at Pentair.  Bastards!
